packages/components/src/config/index.ts

```diff
@@ -1,5 +1,6 @@
 import * as animations from './animations';
 import colors from './colors';
 import * as variables from './variables';
+import * as motionProps from './motion';
 
-export { animations, colors, variables };
+export { animations, colors, variables, motionProps };
```

Comment:
You should avoid reexporting stuff in nested indexes, this is something we want to get rid of completely so adding more stuff here will be added work later.

I would probably suggest renaming `EXPAND` to `export const motionAnimations: { expand: { ...curent stuff in EXPAND } }` and adding also `export motionEasing = { transition: {}, enter: {}, exit: {} }` then exporting both of these const in root index only